src/Donations/Components/ContactsForm.tsx

```diff
@@ -518,10 +537,17 @@ function ContactsForm(): ReactElement {
               <Controller
                 name="companyname"
                 control={control}
-                rules={{ required: true }}
-                render={({ field: { onChange, value } }) => (
+                rules={{
+                  required: t("companyRequired"),
+                  pattern: {
+                    value: /^[\p{L}\p{N}\sß.,'&()!-]+$/u,
+                    message: t("firstNameInvalid"),
```

Comment:
- [ ] Can the error name be changed to `company name is invalid...` rather than `"First name is invalid. Only these special characters are allowed: space ' . -"`
- [ ] I'm able to add `&` in the company name even though the error message doesn't mention it as an accepted special character.

Reply:
Done 754e10708ebee16b94b07578111bd23dc93a7012